docker-bake.hcl

```diff
@@ -1,5 +1,5 @@
 variable "GO_VERSION" {
-    default = "1.20.5"
+    default = "1.20.6"
```

Comment:
We could use `null` value here to default with the one from Dockerfile: https://docs.docker.com/build/bake/reference/#targetargs

Reply:
oh dang; missed your comment; is that supported everywhere now? Want to open a PR?